parking_permits/talpa/order.py

```diff
@@ -21,10 +23,29 @@ class TalpaOrderManager:
     }
 
     @classmethod
-    def _create_item_data(cls, order, order_item):
+    def _get_label(cls, permit, permit_index, has_multiple_permit):
+        registration_number = permit.vehicle.registration_number
+        manufacturer = permit.vehicle.manufacturer
+        model = permit.vehicle.model
+        car_info = f"{registration_number} {manufacturer} {model}"
+        permit_info = f"{permit_index + 1}. Ajoneuvo, "
+        return (permit_info + car_info) if has_multiple_permit else car_info
+
+    @classmethod
+    def _get_product_description(cls, product):
+        start_time = product.start_date.strftime(DATE_FORMAT)
+        end_time = product.start_date.strftime(DATE_FORMAT)
+        return f"{start_time} - {end_time}"
+
+    @classmethod
+    def _create_item_data(cls, order, order_item, permit_index, has_multiple_permit):
```

Comment:
The `permit_index` and `has_multiple_permit` arguments do not seem to fit the method IMO 🤔  They are passed down to `_get_label` via `_create_item_data`. This is a code smell called "tramp data", which can usually be eliminated by choosing an alternative design. Since the `productLabel` is indifferent to order items, we can move the construction one level above (permit level), and pass it to `_create_item_data`.

It might be also good to extract all product-related keys out of this method and put them into a separate method🤔

Reply:
Re-wrote the logic. Thanaks.